include/lsst/meas/base/PixelFlags.h

```diff
@@ -74,8 +80,18 @@ class PixelFlagsAlgorithm : public SimpleAlgorithm {
         CR,
         CR_CENTER,
         BAD,
+        CLIPPED,
         N_FLAGS
     };
+    
+    afw::table::Key<afw::table::Flag> _generalFailure;
+    afw::table::Key<afw::table::Flag> _offImageKey;
+    typedef std::map<std::string, afw::table::Key<afw::table::Flag> > KeyMap;
+    KeyMap _predefinedAnyKeys;
+    KeyMap _predefinedCenterKeys;
+    KeyMap _centerKeys;
+    KeyMap _anyKeys;
+
```

Comment:
upon discussion, you are right, there is no reasons they needed to be public. They have been moved to private, and the key arrays have been merged together. There is now no different predefined and user specified keymaps. The user specified values are simply added